### DISCUSSION TITLE
Application Approved / Denied at the same time

### DISCUSSION THREAD

**Discussion by TechnicDev** [community/community#121696] [GitHub Education]
### Hi everyone,

If I go to https://education.github.com/discount_requests/application I see two status updated. Approved and Denied for today. 
![education.github.com/discount_requests/application](https://github.com/community/community/assets/60631225/03dc62e8-a12c-46f8-88b0-50ad7318023d)
My email only has given a reply for a denied application.

When I open any other page it says my application is pending.
![education.github.com/learner/learn](https://github.com/community/community/assets/60631225/5225ac82-75c8-41df-9240-d1baed204080)
![education.github.com/pack](https://github.com/community/community/assets/60631225/6d87ca73-8f98-4833-8f7e-7918fc13cf2b)

Is there anything I should do right now or should I just continue to wait?
Thanks.

**Comment by gurjeetsinghvirdee** [comment]
Hey @TechnicDev 

Go through this link: https://education.github.com/ & check your benefits
I think you already approved for benefits 

Just ignore the email for now & go through that link

**Reply by TechnicDev** [reply]
When I go to https://education.github.com/pack it shows that blue banner picture I uploaded. Just checked again its still there even after a hard refresh.

**Reply by TechnicDev** [reply]
This happened right after I re-applied to renew my student status.

**Reply by gurjeetsinghvirdee** [reply]
Why are you re-applying? It automatically renews it

**Reply by TechnicDev** [reply]
I got an email telling me to re-apply or I would lose the benefits

**Reply by gurjeetsinghvirdee** [reply]
and now they rejected it, wait some time maybe they will ask for another proof

**Reply by TechnicDev** [reply]
The weird thing is it shows this as well. 
![image](https://github.com/community/community/assets/60631225/17c91050-7640-44d3-8ff0-9744f9b13f30)

I guess I will just wait for now and see what happens.

**Reply by gurjeetsinghvirdee** [reply]
I think github needs update too ☠️

**Reply by TechnicDev** [reply]
For sure, the education portal has been bugging ever since I used it for the first time.

**Reply by gurjeetsinghvirdee** [reply]
Sometimes I don't understand what they really want for proof, they are ok before 2022 but now things are changed and it's frustrating too
